crates/xlinectl/src/command/lease/keep_alive.rs

```diff
@@ -54,14 +54,17 @@ pub(super) async fn execute(client: &mut Client, matches: &ArgMatches) -> Result
 }
 
 /// keep alive forever unless encounter error
-async fn keep_alive_loop(
```

Comment:
There's only one place to call this function, why are you adding `verbose` param?

Reply:
Removed